src/catalog/dependency_manager.cpp

```diff
@@ -22,9 +22,12 @@ void DependencyManager::AddObject(ClientContext &context, CatalogEntry *object,
 			throw InternalException("Dependency has already been deleted?");
 		}
 	}
-	// indexes do not require CASCADE to be dropped, they are simply always dropped along with the table
-	auto dependency_type = object->type == CatalogType::INDEX_ENTRY ? DependencyType::DEPENDENCY_AUTOMATIC
-	                                                                : DependencyType::DEPENDENCY_REGULAR;
+	// indexes/sequences do not require CASCADE to be dropped, they are simply always dropped along with the table
```

Comment:
This block is to ensure that the sequence that is associated with the SERIAL column is always dropped when the parent table is dropped; apparently the SERIAL sequences in Postgres are technically owned by the column (not the table), but I don't think DuckDB has a concept of column-level ownership of things yet.

Reply:
Ooh, maybe I was wrong? https://github.com/duckdb/duckdb/pull/2526

Reply:
Looks good to me.